edx_proctoring/api.py

```diff
@@ -1870,32 +1874,39 @@ def _get_practice_exam_view(exam, context, exam_id, user_id, course_id):
     """
     Returns a rendered view for the practice Exams
     """
+    user = USER_MODEL.objects.get(id=user_id)
+
     student_view_template = None
 
     attempt = get_exam_attempt(exam_id, user_id)
 
     attempt_status = attempt['status'] if attempt else None
 
-    if not attempt_status:
-        student_view_template = 'practice_exam/entrance.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.started:
-        provider = get_backend_provider(exam)
-        if provider.should_block_access_to_exam_material():
-            student_view_template = 'proctored_exam/error_wrong_browser.html'
-        else:
-            # when we're taking the exam we should not override the view
-            return None
-    elif attempt_status in [ProctoredExamStudentAttemptStatus.created,
-                            ProctoredExamStudentAttemptStatus.download_software_clicked]:
-        student_view_template = 'proctored_exam/instructions.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_start:
-        student_view_template = 'proctored_exam/ready_to_start.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.error:
-        student_view_template = 'practice_exam/error.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.submitted:
-        student_view_template = 'practice_exam/submitted.html'
-    elif attempt_status == ProctoredExamStudentAttemptStatus.ready_to_submit:
-        student_view_template = 'proctored_exam/ready_to_submit.html'
+    active_account = user.is_active
+
+    if not active_account:
```

Comment:
I also added this interstitial for practice exams, because I thought that it could be applied here as well, although the ticket only specified onboarding and proctored exams

Reply:
That makes sense. Limited specification is that there's substantially less sensitivity about practice exams, so they're less of concern.